TimberAPI/Dependency/Dependency.cs

```diff
@@ -7,17 +7,19 @@
 using TimberbornAPI.Common;
 using UnityEngine;
 
-namespace TimberbornAPI.Dependency
+namespace TimberbornAPI.DependencySystem
 {
     [HarmonyPatch]
-    public class Dependencies : IDependencies
+    public class Dependency : IDependency
```

Comment:
I think this is an unfortunate name. This class simplifies registering dependencies with Bindito. It's not a dependency itself.
It's like a facade. `BinditoFacade`, `DIFacade` or maybe `DependencyRegistry`?